### ISSUE TITLE
TF serving for TensorFlow 1.7 or more

### ISSUE THREAD

**Issue by FrancescoSaverioZuppichini** [tensorflow/tensorflow#19006]
Dear all,

My machine is a MacBook Pro with  macOS HighSierra 10.13.1 and I am using tf 1.7.  I am trying to use tensorflow serving without any results since the [doc](https://www.tensorflow.org/serving/docker) is not updated for Tensorflow 1.7. The doc says to 

```
git clone --recurse-submodules https://github.com/tensorflow/serving
cd serving/tensorflow
./configure
cd ..
bazel test tensorflow_serving/...
```
But there is no `serving/tensorflow` folder in the container. Following the doc I did:
1) Create the container

```
docker build --pull -t $USER/tensorflow-serving-devel -f Dockerfile.devel .
```
Where the docker file is the one provided by the doc
2) Run it
```
docker run -it $USER/tensorflow-serving-devel
```
3)Try to build tf serving

```
root@5db9a099bd77:/# git clone --recurse-submodules https://github.com/tensorflow/serving
Cloning into 'serving'...

remote: Counting objects: 6476, done.
remote: Compressing objects: 100% (16/16), done.
remote: Total 6476 (delta 7), reused 17 (delta 7), pack-reused 6452
Receiving objects: 100% (6476/6476), 2.43 MiB | 1.07 MiB/s, done.
Resolving deltas: 100% (4628/4628), done.
Checking connectivity... done.
root@5db9a099bd77:/# cd serving/tensorflow
bash: cd: serving/tensorflow: No such file or directory
```

So, basically, how can I run my model in production? I also think that you should maintain a docker image with TF serving already built on it to make it easier for the community to use it.

Cheers,

Francesco Saverio Zuppichini

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by jart** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!

**Comment by Dene33** [comment]
I confirm this. Structure is changed and now I have no idea how to build serving...

**Comment by oscarriddle** [comment]
@FrancescoSaverioZuppichini 
I encountered a similar issue when do the tensorflow-serving 1.7 compilation with a specified python.
#921.
The file structure is changed, and I'm finding a way to pass the include path of python to compilation.
